app.css

```diff
@@ -1,162 +1,167 @@
 .round-right {
```

Comment:
Ignore the `app.css` file for review, the prettier format was missed for this file.